### DISCUSSION TITLE
Two keys in same farmer, how to sync wallet of two keys in same time

### DISCUSSION THREAD

**Discussion by Hugh-Top** [Chia-Network/chia-blockchain#6796] [Q&A]
I add two keys in same farmer, and all harvesters of tow keys connect the farmer to solo。
I need know who rewards of farmer belongs to and the time of reward
now, the chia_wallet only sync one key wallet of last switched
how to sync wallet in same time

**Comment by loppefaaret** [comment]
All the rewards on the same farmer, will go to the same wallet, the address set as "Farmer Reward Target Addresses" in the "Farm" tab of the GUI, or the 2 entries in the `config.yaml` file, `xch_target_address:` is the address _all_ rewards are directed to

**Reply by Hugh-Top** [reply]
think you very much
